Yoast/wordpress-seo
# Yoastseo.js/1809 load morphology data

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

This PR just copies some data from other PR because it needs to be in this repo.
Needed the following for premium implementation:
  * Add shortlink to how to activate a subscription.
  * Exclamation triangle svg. 

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo-premium/pull/2038 for test instructions.

## Quality assurance


Fixes https://github.com/Yoast/YoastSEO.js/issues/1809
